shakirandagire/RecipeSearch
# [Feature #61743860(Sub-Task) Create SearchBar Component

***What does this PR do?***
It adds the search bar for a user to enter search values
***Description of Task to be completed?**
As a user I want to enter search values for recipes in the search bar
***How to test the application***
Clone and set up the project
Use `$ git checkout ft-create-search-bar` to check out the branch containing the changes of this pull request
Use `$ npm start` to start the application
Visit http://localhost:3000/.
***Pivotal Tracker Story***
[#161743860](https://www.pivotaltracker.com/story/show/161743860)
